src/stories/20_Features/Desktop_and_landscape.html

```diff
@@ -0,0 +1,70 @@
+<!---
+
+skipValidation: 'true'
```

Comment:
Is this needed because the feature is still experimental?

Reply:
not really, I noticed all the other story sample are skipping the validation. Do you know why?